Week-2/table1.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Table1</title>
+</head>
+<body>
+    <table border="1px solid black">
```

Comment:
the table tag only accepts integer for border attribute: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/table#attributes
```suggestion
    <table border="1">
```